src/components/auth/postLogin.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import "./postLogin.scss";
+import { ifDev } from "../../utils/removeAttribute";
+
+export default function PostLogin() {
+  return (
+    <div className="container post-login">
```

Comment:
updated class naming convention to match feature branch naming, as well as added more semantic names for sections.